english/the-apocalypse/the-apocalypse.md

```diff
@@ -0,0 +1,3 @@
+Oh no! George R. R. Martin has killed every living person except for you! You have been magically sealed into the deepest dungeon in Westeros!
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.